about/src/css/custom.css

```diff
@@ -1036,6 +1036,7 @@ h4.method-list-item-label .method-list-item-label-name {
   padding: var(--ifm-code-padding-vertical) 0;
 }
 h4.method-list-item-label .method-list-item-label-badge {
+  overflow-wrap: anywhere;
```

Comment:
what does this one do?

Reply:
The jwt example overflows into the sidebar since it is a very long token, unless i add this in